Makefile

```diff
@@ -8,55 +11,45 @@ else
 	TESTEXE := go test ./...
 endif
 
-all: test lint build buildlsp coverage examples ## test, lint, build, coverage test run
+all: lint build buildlsp install examples golden test coverage
```

Comment:
I'm not sure that we should include golden to run on every make (This is because some of the golden.pb files are non deterministic, since google decided to put in random spaces to make it so. This will introduce quite a bit of noise in diffs as a result)